Makefile

```diff
@@ -451,8 +451,8 @@ cervantesupdate: all
 	# remove old package if any
 	rm -f $(CERVANTES_PACKAGE)
 	# Cervantes launching scripts
+	cp $(COMMON_DIR)/spinning_zsync $(INSTALL_DIR)/koreader/spinning_zsync.sh
 	cp $(CERVANTES_DIR)/*.sh $(INSTALL_DIR)/koreader
```

Comment:
I don't think this line covers `platform/cervantes/spinning_zsync` file

Reply:
Nope, I missed the `*.sh` (which I read as `*`) in the CERVANTES one, good catch ^^.